### PR TITLE
Set copyright and package license

### PR DESCRIPTION
**Pull request by tmat**
Prepares repo for change https://github.com/dotnet/arcade/pull/2003 by setting `Copyright` and `PackageLicenseExpression` properties. These values will be required to be set by each repository once https://github.com/dotnet/arcade/pull/2003 is merged.

In order to not break the current builds this change sets the properties conditionally. This condition can be removed once all repos switch to Arcade that has https://github.com/dotnet/arcade/pull/2003.

@markwilkie

### CODE REVIEW

**Comment by tmat** [comment]
@livarcocc

**Comment by tmat** [comment]
@nguerrera @dsplaisted PTAL

File: Directory.Build.props [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
-
   <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
 
+  <PropertyGroup Condition="'$(CopyrightNetFoundation)' != ''">
+    <Copyright>$(CopyrightNetFoundation)</Copyright>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+  </PropertyGroup>
+
   <PropertyGroup>
+    <NoWarn>$(NoWarn);NU5125</NoWarn>
```
Comment (nguerrera) [inline Directory.Build.props]: What is NU5125?

File: Directory.Build.props [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
     <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
     <DebugType>embedded</DebugType>
```
Comment (nguerrera) [inline Directory.Build.props]: Complete aside and existing, but should we be forcing embedded here?

Comment (nguerrera) [reply Directory.Build.props]: @livarcocc I see this was done everywhere in move to arcade. Do you recall the rationale? I thought arcade managed the appropriate symbol type.

File: Directory.Build.props (hunk 1 above)
Comment (tmat) [reply Directory.Build.props]: NeGet warning that we should not use LicenseUrl. Once the Arcade PR is merge we can remove.

**Pull request merged**